docs/en_US/Compression/CompressionReference.rst

```diff
@@ -88,6 +88,9 @@ Pruners
 ..  autoclass:: nni.algorithms.compression.pytorch.pruning.simulated_annealing_pruner.SimulatedAnnealingPruner
```

Comment:
Why are some pruners neither in the one-shot pruner nor in the iterative category?  I think it's a little bit confusing.

Reply:
Actually, some pruners (e.g., `AutoCompressPruner` and `SimulatedAnnealingPruner`) belong to the iterative category. While the length of the code of these pruners is relatively long. So put such pruners in the independent files.